### PR TITLE
Add in hadolint 400X rules

### PR DESCRIPTION
**Pull request by minusworld**

### CODE REVIEW

File: generic/dockerfile/best-practice/maintainer-is-deprecated.yaml
Comment (mschwager) [inline generic/dockerfile/best-practice/maintainer-is-deprecated.yaml:5]: If possible, it might be nice to make this more pointed to avoid false positives. Here it seems like `MAINTAINER` could exist anywhere in the Dockerfile, not necessarily as a command. One option may be `MAINTAINER ...`, but I suspect that has the same issue. Maybe you have better ideas here...

**Review by mschwager — APPROVED** [review]
:tada:

File: generic/dockerfile/best-practice/maintainer-is-deprecated.yaml
Comment (minusworld) [reply generic/dockerfile/best-practice/maintainer-is-deprecated.yaml:5]: I thought that MAINTAINER (in all caps) was the surest way to grab it; once these are all merged I'm going to do a run on MAP for posterity.

The only better option I can think of is doing this as a regex rule and requiring it to be at the beginning of a line. 🤷‍♂️

**Pull request merged**
